.env

```diff
@@ -1 +1 @@
-REACT_APP_SENTRY_URL=https://745cf65377674adbaf85942851bbb125@o267347.ingest.sentry.io/5320754
\ No newline at end of file
+REACT_APP_SENTRY_URL=https://745cf65377674adbaf85942851bbb125@o267347.ingest.sentry.io/5320754
```

Comment:
@ajwhitley Please add this file to the .gitignore you should never check in the .env file

Reply:
Ok, understood.

Reply:
The file has now been removed from the repo and added to the .gitignore file